src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -5291,6 +5291,7 @@ namespace HeatBalanceManager {
 
 				if ( MaxHeatLoadZone( ZoneNum ) > 1.0e-4 ) { // make sure load big enough to divide
 					MaxHeatLoadZone( ZoneNum ) = std::abs( max( MaxHeatLoadZone( ZoneNum ), MinLoad ) );
+					MaxHeatLoadPrevDay( ZoneNum ) = std::abs( max( MaxHeatLoadPrevDay( ZoneNum ), MinLoad ) );
```

Comment:
But this value is set at the bottom of the function and includes the comparison to MinLoad. So what does this actually accomplish?

    // Transfer current daily max and min loads and temperatures to the
    // variables containing the last day's values
    MaxHeatLoadPrevDay( ZoneNum ) = MaxHeatLoadZone( ZoneNum );
    MaxCoolLoadPrevDay( ZoneNum ) = MaxCoolLoadZone( ZoneNum );
    MaxTempPrevDay( ZoneNum ) = MaxTempZone( ZoneNum );
    MinTempPrevDay( ZoneNum ) = MinTempZone( ZoneNum );